docs/docs/Components/components-tools.md

```diff
@@ -257,25 +259,55 @@ This component allows you to call the Serper.dev Google Search API.
 | results | List[Data]| List of search results               |
 | tool    | Tool      | Google Serper search tool for use in LangChain|
 
-## MCP Tools (stdio)
 
-This component connects to a [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction) server over `stdio` and exposes its tools as Langflow tools to be used by an Agent component.
+## MCP Server
 
-To use the MCP stdio component, follow these steps:
+This component connects to a [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction) server and exposes the MCP server's tools as tools for [Agent](/components-agents) in Langflow to use.
 
-1. Add the MCP stdio component to your workflow, and connect it to an agent. The flow looks like this:
+Additional MCP functionality is available through the `/api/v1/mcp` API endpoints. For more information, see [MCP integrations](/integrations-mcp).
```

Comment:
Rather than "additional functionality" just say what it is. 

In addition to being an MCP client that can leverage servers Langflow is also an MCP server that exposes flows as tools, see ...